apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -122,17 +122,21 @@ export default function SectionsSetUpContainer({
     course offerings controller function to populate previousVersionYear and newVersionYear.
     */
     if (isNewSection) {
-      analyticsReporter.sendEvent(EVENTS.COMPLETED_EVENT, {
-        sectionUnitId: section.course?.unitId,
-        sectionCurriculumLocalizedName: section.course?.displayName,
-        sectionCurriculum: section.course?.courseOfferingId, //this is course Offering id
-        sectionCurriculumVersionYear: section.course?.versionYear,
-        sectionGrade: section.grade ? section.grade[0] : null,
-        sectionLockSelection: section.restrictSection,
-        sectionName: section.name,
-        sectionPairProgramSelection: section.pairingAllowed,
-        flowVersion: NEW,
-      });
+      analyticsReporter.sendEvent(
+        EVENTS.COMPLETED_EVENT,
+        {
+          sectionUnitId: section.course?.unitId,
+          sectionCurriculumLocalizedName: section.course?.displayName,
+          sectionCurriculum: section.course?.courseOfferingId, //this is course Offering id
+          sectionCurriculumVersionYear: section.course?.versionYear,
+          sectionGrade: section.grade ? section.grade[0] : null,
+          sectionLockSelection: section.restrictSection,
+          sectionName: section.name,
+          sectionPairProgramSelection: section.pairingAllowed,
+          flowVersion: NEW,
+        },
+        PLATFORMS.BOTH
```

Comment:
This Platform update is the only change here, yeah? And then just formatting?

Reply:
Yep! If you hide whitespace it should hide all the untouched code.